src/lib/menu/menu-directive.ts

```diff
@@ -52,6 +52,7 @@ export class MdMenu implements AfterContentInit, MdMenuPanel, OnDestroy {
 
   @ViewChild(TemplateRef) templateRef: TemplateRef<any>;
   @ContentChildren(MdMenuItem) items: QueryList<MdMenuItem>;
+  @Input('overlapTrigger') overlapTrigger = true;
```

Comment:
tiny nit: if the variable and the html attr have the same name you can omit the alias:
`@Input() overlapTrigger = true;